server.js

```diff
@@ -0,0 +1,21 @@
+require("dotenv").config();
+
+const { createApp } = require("./app");
+const  AppDataSource  = require("./src/models/data-source");
```

Comment:
```suggestion
const  appDataSource  = require("./src/models/data-source");
```
자바스크립트에서는 클래스나 enum같은 경우 외에는, 즉 지금 처럼 instance같은 경우에는 대문자로 시작하지 않는 것이 일반적인 컨벤션입니다-!